migrations/2_deploy.js

```diff
@@ -1,25 +1,40 @@
 const UnipoolFactory = artifacts.require('./UnipoolFactory.sol');
 const Unipool = artifacts.require('./Unipool.sol');
 const UnipoolMock = artifacts.require('./UnipoolMock.sol');
+const UnipoolForeignPairMock = artifacts.require('./UnipoolMock.sol');
 const HoneyTokenMock = artifacts.require('./HoneyTokenMock.sol');
-const UniswapTokenMock = artifacts.require('./UniswapTokenMock.sol');
+const HoneyTokenActual = '0x71850b7E9Ee3f13Ab46d67167341E4bDc905Eef9';
```

Comment:
This is an address so should be referred to as such. Global constants in scripts are labelled with caps and underscores eg: `HONEY_TOKEN_XDAI_ADDRESS`